recipe-server/normandy/recipes/urls.py

```diff
@@ -28,7 +28,7 @@
     url(r'^api/v2/', include(v2_router.urls)),
     url(r'^api/v1/', include(v1_router.urls)),
     url(
-        r'^api/v1/action/(?P<name>[_\-\w]+)/implementation/(?P<impl_hash>[0-9a-f]{40})/$',
+        r'^api/v1/action/(?P<name>[_\-\w]+)/implementation/(?P<impl_hash>.*)/$',
```

Comment:
Would this match slashes as well? We probably want to keep this limited to base64 symbols.